### DISCUSSION TITLE
Error: The input tensor size is not equal to the model input type

### DISCUSSION THREAD

**Discussion by yktyagi** [openvinotoolkit/openvino#25265] [Q&A]
I was trying to run a classification sample with openvino:

python3 classification_sample_async.py -m public/googlenet-v3/FP32/googlenet-v3.xml -i banana.jpg -d GPU

and getting this error:

super().start_async(
RuntimeError: Exception from src/inference/src/cpp/infer_request.cpp:235:
Check 'is_dynamic || port.get_shape() == tensor->get_shape()' failed at src/inference/src/dev/isync_infer_request.cpp:277:
The input tensor size is not equal to the model input type: got [1,299,3,3] expecting [1,299,3,299].

I got the model in follwoing way:

omz_downloader --name googlenet-v3

omz_converter --name googlenet-v3

I am using openvino 2024.2.0 on Ubuntu 22.04

**Comment by ilya-lavrenov** [comment]
CC @Wovchena

**Comment by Wovchena** [comment]
Yep. The sample expects the model to have NCHW layout https://github.com/openvinotoolkit/openvino/blob/0f2b44b352f664bcfa3d042a3739f1f0be3160f6/samples/python/classification_sample_async/classification_sample_async.py#L89 and can't process any other layout. You can use https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/classification_demo/python. It handles NCHW and NHWC.
